plasmapy/formulary/collisions.py

```diff
@@ -2067,6 +2067,7 @@ def coupling_parameter(
 
 
 class CollisionFrequencies:
+    @particles.particle_input
     @validate_quantities(
```

Comment:
I changed to order of these decorators because, for instance methods, `particle_input` currently has to be the outer decorator.  I plan on creating an issue after this is merged to remove this limitation, since I'm not sure if this will require changes in `particle_input` or `validate_quantities`.